rskj-core/src/fuzzTest/fuzzing.gradle

```diff
@@ -0,0 +1,591 @@
+    classpath = sourceSets.fuzzTest.runtimeClasspath
+    jvmArgs '-Xss32m', '-Xmx3G'
+}
+
+task RLPFuzzEncodeDecodeString(type: Test) {
+    useJUnitPlatform {
+        includeTags 'RLPFuzzEncodeDecodeString'
+    }
+    testClassesDirs = sourceSets.fuzzTest.output.classesDirs
+    classpath = sourceSets.fuzzTest.runtimeClasspath
+    jvmArgs '-Xss32m', '-Xmx3G'
+}
+
+task RLPFuzzEncodeDecodeElement(type: Test) {
+    useJUnitPlatform {
+        includeTags 'RLPFuzzEncodeDecodeElement'
+    }
+    testClassesDirs = sourceSets.fuzzTest.output.classesDirs
+    classpath = sourceSets.fuzzTest.runtimeClasspath
+    jvmArgs '-Xss32m', '-Xmx3G'
+}
+
+// DB
+task MutableTrieCacheFuzzPuts(type: Test) {
+    useJUnitPlatform {
+        includeTags 'MutableTrieCacheFuzzPuts'
+    }
+    testClassesDirs = sourceSets.fuzzTest.output.classesDirs
+    classpath = sourceSets.fuzzTest.runtimeClasspath
+    jvmArgs '-Xss32m', '-Xmx3G'
+}
+
+task RepositoryFuzzStorageRoot(type: Test) {
+    useJUnitPlatform {
+        includeTags 'RepositoryFuzzStorageRoot'
+    }
+    testClassesDirs = sourceSets.fuzzTest.output.classesDirs
+    classpath = sourceSets.fuzzTest.runtimeClasspath
+    jvmArgs '-Xss32m', '-Xmx3G'
+}
+
+/** [FUZZ TARGETS]
+ * Runtime:
+ *          orgEthereumVMFuzzOpcodes, coRSKVMFuzzTestNativeContractInputSize, WorldFuzzWorldFuzz, PrecompiledContractFuzzIdentityRandom,
+ *          PrecompiledContractFuzzSha256Random, PrecompiledContractFuzzRimpempdRandom, PrecompiledContractFuzzEcrecoverRandom,
+ *          PrecompiledContractFuzzModexpRandom, ExtractPublicKeyFromExtendedPublicKeyFuzzExecuteRandom, GetMultisigScriptHashRandomFuzzExecuteMixed,
+ *          DeriveExtendedPublicKeyRandomFuzzExecuteRandom, Secp256k1ServiceNativeFuzzRecoverFromSignatureRecId, Secp256k1ServiceNativeFuzzRecoverFromSignatureDirectly,
+ *          AltBN128FuzzAdd, AltBN128FuzzMul, AltBN128FuzzPairing,
+ * Discovery:
+ *          DiscoveryPacketDecoderFuzzDecode, DiscoveryPacketDecoderFuzzDecode2, DiscoveryPacketDecoderFuzzInput, MessageCodecFuzzMessageCodec,
+ * Misc:
+ *          ByteUtilFuzzBUFuzz,
+ * Core:
+ *          BlockDecodingFuzzBlockDecoding, CallContractFuzzContractReturningOne,
+ * RLP:
+ *          RLPFuzzDecodeInt, RLPFuzzDecodeEncode, RLPFuzzDecodeBigInteger, RLPFuzzGetNextElementIndex, RLPFuzzFullTraverse, RLPFuzzDecode2,
+ *          RLPFuzzFirstElement, RLPFuzzDecodeList, RLPFuzzDecode2OneItem, RLPFuzzEncodeLength, RLPFuzzEncodeByte, RLPFuzzEncodeShort, RLPFuzzEncodeInt,
+ *          RLPFuzzEncodeString, RLPFuzzEncodeElement, RLPFuzzEncodeListHeader, RLPFuzzEncodeSet, RLPFuzzEncodeList, RLPFuzzEncodeDecodeByte,
+ *          RLPFuzzEncodeDecodeShort, RLPFuzzEncodeDecodeInt, RLPFuzzEncodeDecodeString, RLPFuzzEncodeDecodeElement,
+ * DB:
+ *          MutableTrieCacheFuzzPuts, RepositoryFuzzStorageRoot
+ **/
+task runAllFuzzTests {
+    dependsOn orgEthereumVMFuzzOpcodes, coRSKVMFuzzTestNativeContractInputSize, WorldFuzzWorldFuzz,
```

Comment:
just a note.. due to dependsOn, if any test fails, coverage report won't be generated.